lib/dish.rb

```diff
@@ -0,0 +1,9 @@
+class Dish
+  attr_reader :name, :price, :item
+
+  def initialize(name, price)
+    @name = name
+    @price = price
+    @item = { :name => @name, :price => @price }
+  end
```

Comment:
Nice to have factored this out into its own class - was the intention for the menu class to reference it?